Source/JavaScriptCore/inspector/remote/RemoteInspector.h

```diff
@@ -301,8 +306,7 @@ class JS_EXPORT_PRIVATE RemoteInspector final
 #endif
     bool m_shouldSendParentProcessInformation { false };
     bool m_automaticInspectionEnabled { false };
-    bool m_automaticInspectionPaused { false };
-    TargetID m_automaticInspectionCandidateTargetIdentifier { 0 };
+    HashSet<TargetID, WTF::IntHash<unsigned>, WTF::UnsignedWithZeroKeyHashTraits<unsigned>> m_pausedAutomaticInspectionCandidates;
```

Comment:
I wonder if we can use `WTF_GUARDED_BY_LOCK(m_mutex)` so the compiler can ensure this set is guarded?